sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -473,7 +482,6 @@ static const struct snd_soc_ops sdw_ops = {
 	.shutdown = sdw_shutdown,
 };
 
-static void rt711_init(const struct snd_soc_acpi_link_adr *link,
```

Comment:
something's missing here, this will not compile

Reply:
sorry, it is caused by merge conflict ion. I update PR now